sormas-backend/src/main/java/de/symeda/sormas/backend/externalsurveillancetool/ExternalSurveillanceToolGatewayFacadeEjb.java

```diff
@@ -60,15 +66,32 @@ public class ExternalSurveillanceToolGatewayFacadeEjb implements ExternalSurveil
 	private EventService eventService;
 	@EJB
 	private ExternalShareInfoService shareInfoService;
+	@EJB
+	private UserService userService;
 
 	@Override
 	@AuditIgnore
+	@PermitAll
 	public boolean isFeatureEnabled() {
 		return configFacade.isExternalSurveillanceToolGatewayConfigured();
 	}
 
 	@Override
-	public void sendCases(List<String> caseUuids, boolean archived) throws ExternalSurveillanceToolException {
+	@RightsAllowed(UserRight._EXTERNAL_SURVEILLANCE_SHARE)
+	public void sendCases(List<String> caseUuids) throws ExternalSurveillanceToolException {
+		doSendCases(caseUuids, false);
```

Comment:
This method now is used only in the ui module, where it was always called with "archived = false", so there is no need to have the archived argument.

It can be changed when needed.